rust/README.md

```diff
@@ -1,36 +1,75 @@
 <p align="center"><a href="https://cube.dev"><img src="https://i.imgur.com/zYHXm4o.png" alt="Cube.js" width="300px"></a></p>
 
-[Website](https://cube.dev) • [Docs](https://cube.dev/docs) • [Examples](#examples) • [Blog](https://cube.dev/blog) • [Slack](https://slack.cube.dev) • [Twitter](https://twitter.com/thecubejs)
+[Website](https://cube.dev) • [Docs](https://cube.dev/docs) •
+[Examples](#examples) • [Blog](https://cube.dev/blog) •
+[Slack](https://slack.cube.dev) • [Twitter](https://twitter.com/thecubejs)
 
 [![npm version](https://badge.fury.io/js/%40cubejs-backend%2Fserver.svg)](https://badge.fury.io/js/%40cubejs-backend%2Fserver)
 [![GitHub Actions](https://github.com/cube-js/cube.js/workflows/Rust/badge.svg)](https://github.com/cube-js/cube.js/actions?query=workflow%3ARust+branch%3Amaster)
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Fcube-js%2Fcube.js.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Fcube-js%2Fcube.js?ref=badge_shield)
 
-
-Cube Store
-==========
+# Cube Store
```

Comment:
This file is uploaded to Docker Hub, maybe it's better to introduce a new section inside our documentation?

Reply:
/cc @keydunov

Reply:
/cc @keydunov

Reply:
Maybe Docker readme should be a different file? wdyt @hassankhan @ovr

Reply:
Could be, I'm not opposed to it. Might also be good to get @paveltiunov's thoughts too 😄

Reply:
Let's just get this one merged for now.